mantidproject/mslice
# New DataLoader widget

New widget for loading workspaces. Appears on the main 'Data Loading' tab and includes a file browser.

**To test:**
- Navigate to a workspace file and verify that the workspace appears in the workspace manager after hitting the 'Load Data' button. 
- Select multiple files and press 'Load Data'. Check each workspace appears in the workspace manager individually.
- Select multiple files and press 'Load and Merge Data'. Check a single `_merged` workspace appears in the workspace manager.
- Also check the Refresh/Back buttons with the path textbox and error handling 

Fixes #206
